packages/cli/lib/validators/marketplaceValidators/templateLimitValidator.js

```diff
@@ -0,0 +1,26 @@
+const { walk } = require('@hubspot/cli-lib/lib/walk');
+const { ERROR_SEVERITY } = require('../constants');
+
+const VALIDATOR_NAME = 'TemplateLimitValidator';
+const TEMPLATE_LIMIT = 50;
+const TEMPLATE_REGEX = new RegExp(/.html+$/);
+
+// Validate that the theme does not contain more than TEMPLATE_LIMIT templates
+function templateLimitValidator(absoluteThemePath) {
```

Comment:
I realize that these are WIP and you mention correctness so maybe they are planned, but I'd recommend adding unit tests for the validators.

Reply:
I added tests for the validators in 1a1eeec . I'm still not 100% sure that they're validating the right criteria, but I'll confirm before merging.